.github/workflows/genai_python_lib.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           echo "$models" | while read -r model_name model_path; do
               optimum-cli export openvino --trust-remote-code --weight-format fp16 --model "$model_name" "$model_path"
           done
-          python -m pytest test_generate_api.py
+          GENAI_BUILD_DIR=../../build python -m pytest test_generate_api.py
```

Comment:
tests are example of user code.
users should not know about tokenizers, etc and that some vars need to set, except `PYTHONPATH`